web/server/codechecker_server/api/report_server.py

```diff
@@ -947,6 +1003,13 @@ def getRunResults(self, run_ids, limit, offset, sort_types,
 
                 q = q.limit(limit).offset(offset)
 
+                report_details = {}
+
+                # Get report details if it is required.
+                if get_details:
+                    report_ids = [r[1] for r in q]
```

Comment:
We are querying a lot of information here and we keep the report_id only.
In the next loop we query the same information again.

Would it be much slower if we would query the report details for each report (one by one) in the for loop below?